pkg/scheduler/scheduler_test.go

```diff
@@ -2790,8 +2791,17 @@ func TestResourcesToReserve(t *testing.T) {
 			if err != nil {
 				t.Errorf("Error when adding ClusterQueue to the cache: %v", err)
 			}
-			cachedCQ := cqCache.Snapshot().ClusterQueues["cq"]
-			cachedCQ.Usage = tc.cqUsage
+			snapshot := cqCache.Snapshot()
```

Comment:
QQ does it make sense to go a step further and add the workloads to the cache, and only then take the snapshot?  I suspect this might be closer to what happens in the production code.

Reply:
Done.